docs/pages/enterprise/sso.mdx

```diff
@@ -335,8 +343,10 @@ must be able to:
 
 If something is not working, we recommend to:
 
+- Run the connector definition through [`tctl sso test`](../setup/reference/cli.mdx#tctl-sso-test) command.
```

Comment:
I'd suggest removing the link and adding an example of how to use `tctl sso test` to troubleshoot.